cpp/test/stats/weighted_mean.cu

```diff
@@ -73,18 +94,18 @@ class RowWeightedMeanTest : public ::testing::TestWithParam<WeightedMeanInputs<T
     auto stream = handle.get_stream();
     // device-side data
     din.resize(len);
-    dweights.resize(cols);
-    dexp.resize(rows);
-    dact.resize(rows);
+    dweights.resize(rows);
```

Comment:
I would take `RowWeightedMean` to be that we want to do the mean along rows, right? That would mean we want the weights to be sized to the cols so we do the weighted average of each row, right? So the final array produced would be the number of rows when along rows == true and the number of columns when along rows == false.